examples/CMakeLists.txt

```diff
@@ -149,6 +149,7 @@ foreach(EXAMPLE
   57_hopper_grouped_gemm
   58_ada_fp8_gemm
   59_ampere_gather_scatter_conv
+  sycl
```

Comment:
Following their examples, I suppose we could have something along the lines of `60_PVC_bfloat16_gemm_tiled`, rather than keeping  it in a SYCL and then a PVC folder

Reply:
They have created a `cute` folder in the same `examples` folder, which is what we wanted to mimic with `sycl` here.